Tests/FuzzilliTests/MutationsTest.swift

```diff
@@ -68,16 +77,20 @@ class MutationsTests: XCTestCase {
         let v3 = b.binary(b.loadInt(1), b.loadInt(2), with: .Add)
         b.unary(.BitwiseNot, v3)
         let program = b.finalize()
-        var types: [[Int: Type]?] = [[0: .string], [1: .integer], [2: .integer], [3: .integer], [4: .integer]]
-        program.runtimeTypes = VariableMap<[Int: Type]>(types)
+        var types: [[TypeIndexPair]?] = [
+            [TypeIndexPair(index: 0, type: .string)], [TypeIndexPair(index: 1, type: .integer)],
```

Comment:
I think we should have a convenience constructor for `ProgramTypes` for the (common - at least in tests) case that every variable only has a single type (at the point where it is defined).
That probably requires determining when the variable is defined, which can by done by the [VariableAnalyzer](https://github.com/googleprojectzero/fuzzilli/blob/759e3419159cd1a746f5253b56a99108f673a314/Sources/Fuzzilli/FuzzIL/Analyzer.swift#L62)

Reply:
yeah we can have some constructor to make it easier, but analyzer is tied with program and this structure is not

Reply:
The analyzer should also be fine with just `Code` (I guess we just need to change the constructor or add a new one), but yes, we can also remove the name "Program" from this structure, don't have a strong preference :)